### DISCUSSION TITLE
Cannot cache routes when configuring update endpoint

### DISCUSSION THREAD

**Discussion by origjinali** [livewire/livewire#6888] [7. Bugs (archived)]
### Livewire version

v3.0.5

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Install clean Laravel project
2. Install Livewire v3
3. Set custom update endpoint in boot function of AppServiceProvider:
```
Livewire::setUpdateRoute(function ($handle) {
    return Route::post('/endpoint/update', $handle);
});
```
4. Try to cache routes with `php artisan route:cache`

### Additional information

Presented error:

```
In AbstractRouteCollection.php line 247:
                                                                               
  Unable to prepare route [endpoint/update] for serialization. Another route has   
  already been assigned name [livewire.update]. 
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by cdarken** [comment]
This might be too late, but in case you're still wondering or for anybody else trying the same thing, the fix is renaming your route like this:
```
Livewire::setUpdateRoute(function ($handle) {
    return Route::post('/endpoint/update', $handle)->name('app.livewire.update');
});
```
It's important that the route name ends with `livewire.update` otherwise it will receive the same name as the original route.
